src/Data/Foreign/Generic.purs

```diff
@@ -46,12 +46,17 @@ defaultOptions =
   , unwrapNewtypes: false
   , unwrapSingleArgumentConstructors: true
   , maybeAsNull: true
+  , tupleAsPair: false
```

Comment:
Would `tupleAsArray` be a better name for this option? Since there is no concept of `pair` in JS?